public/js/data.js

```diff
@@ -436,21 +436,19 @@ function generateImageForOnline() {
 // Function to save data online
 function save() {
     projectSaved = true;
-
-    $('.loadingIcon').fadeIn();
```

Comment:
Move this inside the else statement, not remove it.

Reply:
Fixed in my recent commit 👍

Reply:
you're welcome.